src/main/java/me/moodcat/eemcsdata/parser/ParseData.java

```diff
@@ -51,18 +51,17 @@ public JsonData getResult() {
      *            - the path to the input file
      * @param output
      *            - the path where to output the file.
-     * @return A json data class to store the data. Returns null if an I/O exception occured.
+     * @return A json data class to store the data. Returns null if an I/O exception occurred.
+     * @throws IOException
+     *             If the file can not be found,throw IOException.
      */
-    public JsonData parseFile(final String input, final String output) {
```

Comment:
Could you change this method to be able to read from an inputstream? It's good practise to access resources from the JAR instead of the system file path (because the file path is not existing when running from the built jar). 

```java
 try(InputStream in = SongDataTest.class.getResourceAsStream("/acousticbrainz/testData/32098913.json")) {
            AcousticBrainzData features = objectMapper.readValue(in, AcousticBrainzData.class);
/// ....
        }
```